mission-lotto/src/main/java/com/web/lotto/domain/exception/NotSetWinningNumbers.java

```diff
@@ -0,0 +1,8 @@
+package com.web.lotto.domain.exception;
+
+public class NotSetWinningNumbers extends IllegalStateException{
```

Comment:
```suggestion
public class NotSetWinningNumbers extends IllegalStateException {
```

CustomException에서 IllegalStateException을 상속하신 이유가 무엇인가요?
스프링 커스텀 예외에 대해 학습하고 수정해주세요!